### ISSUE TITLE
Add Accessibility Info Input for Bookings

### ISSUE THREAD

**Issue by atoff** [BristolSTA/uobtheatre-web#320]
- In the booking purchase flow, add an input to allow user to indicate any accessibility concerns (wheelchair, epilepsy, etc.)
- In the admin booking view, display any accessibility concern when viewing a booking
- In the admin bookings view, allow for filtering to find bookings with accessibility concerns

See https://github.com/BristolSTA/uobtheatre-api/blob/dev/schema.graphql#L25

**Comment by HappyNTH** [comment]
Changing this to a yes/no boolean for requiring a wheelchair space, since we can't do anything meaningful with any other information. Other access issues directed to email support@uobtheatre.com

Some way to alert show crew when a wheelchair booking is made would also be useful.

**Comment by atoff** [comment]
> Changing this to a yes/no boolean for requiring a wheelchair space, since we can't do anything meaningful with any other information. Other access issues directed to email [support@uobtheatre.com](mailto:support@uobtheatre.com)
> 
> Some way to alert show crew when a wheelchair booking is made would also be useful.

I'd tend to disagree about making this a boolean. In the past we have had people talk about mobility problems etc, so we have reserved seats that are more accessible for them. Need to remember that this is not just for the winston as well, also external venues and or anson rooms, and also that external production teams might still be interested in this information, I don't think we should take the decision for them

**Comment by HappyNTH** [comment]
> I'd tend to disagree about making this a boolean. In the past we have had people talk about mobility problems etc, so we have reserved seats that are more accessible for them. Need to remember that this is not just for the winston as well, also external venues and or anson rooms, and also that external production teams might still be interested in this information, I don't think we should take the decision for them

I agree, but I think in that case we should turn it more towards _access_ rather than _accessibility_? I also have concerns about how we then expose the access/ibility information, since it verges on medical-type at times and could be pretty sensitive

> Also, when a booking is completed, if it has accessibility information this is already automatically emailed to people with access to the production https://github.com/BristolSTA/uobtheatre-api/blob/dev/uobtheatre/bookings/models.py#L553

That's fucking cool

**Comment by atoff** [comment]
I think what nomenclature we use is up to us. I think with appropriate warnings in the interface about how this information will be used / who it is sent to it should be fine - they shouldn't be including information they would consider sensitive. Potentially need to add a section in the privacy policy about it.

**Comment by HappyNTH** [comment]
It may be useful to have some canned responses too: they could be checkboxes that just append something like `Wheelchair space requested` to Consideration of external venues is a good point: in whatever warning we include about how the information will be used (sent to the production team) we should indicate that we can't guarantee anything. Also should make it clear in the email that gets sent to prod teams that they should speak to venue staff (STA/whatever) about accommodating and respond by email if something isn't possible.
